kickstarter/android-oss
# NTV-591: Android 13

# 📲 What

Targeting API 33

# 🤔 Why
- Feature branch contains all changes needed to support Android 13 notification permissions for KS notifications and Braze notifications

|  |  |

# 📋 QA

- Install in an Android 13 device,
- make sure the app asks for notification permissions 
- make sure you get push notifications

# Story 📖

[NTV-591](https://kickstarter.atlassian.net/browse/NTV-591)
